aws-test/bin

```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/terraform/0.15.3/bin
```

Comment:
@jzendle Can you please remove this file?

Reply:
done. Thanks!